src/main/java/mServer/crawler/sender/ard/ArdCrawler.java

```diff
@@ -42,6 +42,11 @@ protected synchronized void meldungThreadUndFertig() {
       mlibFilmeSuchen.meldenFertig(Const.HR);
       mlibFilmeSuchen.meldenFertig(Const.BR);
       mlibFilmeSuchen.meldenFertig("rbtv");
+      mlibFilmeSuchen.meldenFertig("ONE");
+      mlibFilmeSuchen.meldenFertig("ARD-alpha");
+      mlibFilmeSuchen.meldenFertig("Funk.net");
+      mlibFilmeSuchen.meldenFertig(Const.SR);
```

Comment:
für SR+PHOENIX gibt es im Master eigene Crawler.
werden dadurch nicht Filme mehrfach aufgelistet, weil ARD andere Urls hat in seiner Mediathek?

Reply:
Bei PHOENIX liefert ARD andere Titel und oft die Teilstücke zu den Sendungen (z.B. phoenix der tag vom 14.05.2024 vs u.a. LIVE - Schaltgespräch mit...). Grundsätzlich betrift das aber nur 10% der Sendungen und wir dann nur besser durch den Einheitlichen Sender.
Bei SR sind heute viele Sendungen doppelt (SR&ARD). Ausserdem gibt es Sendungen mit unterschiedlichem Titel aber gleichem Inhalt. Da kann man später mal prüfen ob es überhautp etwas gibt was nicht in der ARD schon drin ist.